src/pages/VolunteersJobListingFS.page

```diff
@@ -159,6 +168,7 @@
                     	if (x == '') x = JobShiftDateTimeString('{!shift.Start_Date_Time__c}', '{!shift.Duration__c}');
                         OpenSignUpDlg(event.clientX, event.clientY, '{!job.Id}', '{!shift.Id}', '{!JSENCODE(job.Name)}', x); 
                         return false;"
+                        style="{!IF(job.External_Signup_Url__c == null, '', 'display: none;')}"
```

Comment:
@daizylogik I was hoping you would also move this display:none logic to just be part of the rendered statement on the outputPanel.